csrankings-r.csv

```diff
@@ -1090,6 +1090,7 @@ Roch Guérin,Washington University in St. Louis,http://www.cse.wustl.edu/~guerin
 Rocky Slavin,University of Texas at San Antonio,http://galadriel.cs.utsa.edu/~rslavin,Y4c96KkAAAAJ
 Roded Sharan,Tel Aviv University,http://www.cs.tau.ac.il/~roded,64G5UgMAAAAJ
 Roderic A. Grupen,University of Massachusetts Amherst,https://www.cics.umass.edu/faculty/directory/grupen_roderic,NOSCHOLARPAGE
+Roderich Groß,TU Darmstadt,https://www.informatik.tu-darmstadt.de/fb20/organisation_fb20/professuren_und_gruppenleitungen/index.en.jsp,s-Unqd8AAAAJ

```

Comment:
Needs alias entry as well ("Roderich Gross").

Reply:
How do I create the alias entry? Adding him to dblp-aliases.csv seems not possible as I do not have the privileges!

Reply:
You need two separate entries, one for each DBLP name.